File: goormlevel/175177/1. 거스름-돈/거스름-돈.py
import sys

input = sys.stdin.readline
write = sys.stdout.write

def solve():
	n = int(input().rstrip())
	if not n: return

	nlist = [40, 20, 10, 5, 1]

	count = 0
	for i in nlist:
		count += n // i
		n %= i

	write(str(count) + "\n")

if __name__ == "__main__":
	solve()

File: goormlevel/175177/1. 거스름-돈/거스름-돈.c
#include <stdio.h>

int main() {
	int n = 0;
	if (scanf("%d", &n) != 1) return 0;

	int nlist[5] = {40, 20, 10, 5, 1};

	int count = 0;
	for (int i = 0; i < 5; i++) {
		count += n / nlist[i];
		n %= nlist[i];
	}

	printf("%d\n", count);

	return 0;
}
